api/test_app.py

```diff
@@ -0,0 +1,101 @@
+"""Tests for get all entries"""
+import json
+import unittest
+
+
+from api import app
+
+"""Entries Request Response Tests"""
+
+
+class TestEndPoints(unittest.TestCase):
+    """Set Defaults"""
+
+    def setUp(self):
+        self.my_diary = app.app.test_client()
+        self.path = '/api/v1'
+
+    # The 4 main tests
+    def test_receive_entries(self):
```

Comment:
Its good practice to name tests after the name of the function being tested or at least an implication of it. ie. This test would have been better if named ` def test_get_entries(self)`

Reply:
All the mentioned issues are now being fixed thanks to Your Feedback.
Thank You For Your Support. :smiley: